allensdk/mouse_connectivity/grid/_schemas.py

```diff
@@ -80,4 +80,4 @@ class OutputSchema(RaisingSchema):
 
 
 class OutputParameters(OutputSchema): 
-    output_file_paths = Dict(required=True)
+    output_file_paths = List(String, required=True)
```

Comment:
A list of strings certainly makes more sense, but I'm assuming everything downstream of this (if any) is okay with the output suddenly changing from the originally specified `dict`?

Or is it the case that the `Dict` specification was just never checked by the older version of Argschema?

Reply:
@njmei It was not checked by the older version of arschema. Here's an example output json, where you can see it being written as a list of strings: /allen/programs/celltypes/production/mousegenetictools/prod49/image_series_1092407404/202103292256_TISSUECYTE_GRID_CLASSIC_QUEUE_1092407404_output.json